lib/js/styles/ColorsThemes/ColorsThemes.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+	<div class="sb__presentation">
+		<div class="sb__row">
+			<ol class="sb__list">
+				<li class="sb__legend">Theme WNL</li>
+				<li class="sb__legend">Theme Bodywork</li>
+			</ol>
+		</div>
+
+		<div v-for="(colors, colorWNLCategory) in WNLThemeWNLColors" :key="colorWNLCategory" class="sb__row">
+			<h2>Theme WNL</h2>
+			<div v-for="color in colors" :key="color.id" class='sb__table'>
+				<div class="sb__tableId">{{ color.label }}</div>
+				<div class="sb__tableValue">{{ color.value }}</div>
+				<div class="sb__tableColor">
+					<span class="sb__color" :style="{ background: color.value }"></span>
+				</div>
+			</div>
+		</div>
+
+		<div v-for="(colors, colorBodyworkCategory) in BodyworkThemeColors" :key="colorBodyworkCategory" class="sb__row">
+			<h2>Theme Bodywork</h2>
+			<div v-for="color in colors" :key="color.id" class='sb__table'>
+				<div class="sb__tableId">{{ color.label }}</div>
+				<div class="sb__tableValue">{{ color.value }}</div>
+				<div class="sb__tableColor">
+					<span class="sb__color" :style="{ background: color.value }"></span>
+				</div>
+			</div>
+		</div>
+
+	</div>
+</template>
+
+
+<script lang="ts">
+
+import themeWNL from '../../../styles/settings/colors/_raw.json'
+import themeBodywork from '../../../styles/settings/colors/_raw-bodywork.json'
+
+export default {
+	name: 'ColorsThemes',
```

Comment:
this component uses styles defined in another component, it's almost identical as `Colors.vue`. I think we should consider changes sugested in https://github.com/bethinkpl/design-system/pull/98/files#r822549545 to address this problems

Reply:
instead of moving styles to separate scss file, maybe we could reuse component? By doing that we avoid HTML markup duplication

Reply:
It's done.